### PR TITLE
Adding Hypervisor Manager Files to API 1000 and API 1200

### PR DESCRIPTION
**Pull request by AsisBagga**
### Description
Added Hypervisor Manager resources to API-1000 and API-1200

### Issues Resolved
N/A

### Check List
- [ ] New functionality includes testing.
  - [ yes ] All tests pass (`$ rake test`).
- [ ] New functionality has been documented in the README if applicable.
  - [ ] New functionality has been thoroughly documented in the examples (please include helpful comments).
- [ yes ] New endpoints supported are updated in the endpoints-support.md file.
- [ yes ] Changes are documented in the CHANGELOG.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -10,6 +10,7 @@ This release does not support any new features added in API versions 800,1000 an
 - Golden Image
 - OS Volume
 - Plan Script
+- Hypervisor Manager
```
Comment (soodpr) [inline CHANGELOG.md:13]: please add it in a new unreleased version as this has been already released

File: examples/_client.rb.example [hunk 2]
```diff
@@ -2,10 +2,10 @@ require_relative './../lib/oneview-sdk'
 require 'pry'
 
 @client = OneviewSDK::Client.new(
-  url: 'https://oneview.example.com', # or set ENV['ONEVIEWSDK_URL']
+	url: 'https://oneview.example.com', # or set ENV['ONEVIEWSDK_URL']
   user: 'Administrator',  # or set ENV['ONEVIEWSDK_USER']
   password: 'secret123',  # or set ENV['ONEVIEWSDK_PASSWORD']
-  ssl_enabled: false
+  ssl_enabled: false,
```
Comment (soodpr) [inline examples/_client.rb.example:8]: please remove these changes which are not required.

File: examples/shared_samples/hypervisor_manager.rb [hunk 3]
```diff
@@ -22,6 +22,10 @@
 # api_version = 800 & variant = C7000 to OneviewSDK::API800::C7000::HypervisorManager
 # api_version = 800 & variant = Synergy to OneviewSDK::API800::Synergy::HypervisorManager
 
+@hypervisor_manager_ip= "172.18.13.11"
```
Comment (soodpr) [inline examples/shared_samples/hypervisor_manager.rb:25]: this is already configured in _client.rb.example file

File: examples/shared_samples/hypervisor_manager.rb [hunk 4]
```diff
@@ -22,6 +22,10 @@
 # api_version = 800 & variant = C7000 to OneviewSDK::API800::C7000::HypervisorManager
 # api_version = 800 & variant = Synergy to OneviewSDK::API800::Synergy::HypervisorManager
```
Comment (chebroluharika) [inline examples/shared_samples/hypervisor_manager.rb:23]: Add API1000, API1200 for synergy and c7000 under supportedAPI's section and in the section present below it.

File: CHANGELOG.md (hunk 1 above)
Comment (AsisBagga) [reply CHANGELOG.md:21]: @soodpr Changes Updated

File: examples/_client.rb.example (hunk 2 above)
Comment (AsisBagga) [reply examples/_client.rb.example]: @soodpr Changes Updated

File: examples/shared_samples/hypervisor_manager.rb (hunk 3 above)
Comment (AsisBagga) [reply examples/shared_samples/hypervisor_manager.rb]: @soodpr Changes Updated

File: examples/shared_samples/hypervisor_manager.rb (hunk 4 above)
Comment (AsisBagga) [reply examples/shared_samples/hypervisor_manager.rb:27]: @chebroluharika Changes Updated

File: CHANGELOG.md (hunk 1 above)
Comment (soodpr) [reply CHANGELOG.md:21]: its still present

Comment (AsisBagga) [reply CHANGELOG.md]: Its removed now, please check.

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,15 +1,15 @@
 ## v5.11.0(unreleased)
 
 #### Notes
-Extended support to Hypervisor Cluster Profile Rest API version 1000 and API version 1200.
+Extended support to Hypervisor Manaager and Hypervisor Cluster Profile Rest API version 1000, and  API version 1200.
```
Comment (chebroluharika) [inline CHANGELOG.md:4]: It should be Hypervisor Manager

Comment (AsisBagga) [reply CHANGELOG.md]: @chebroluharika updated.

**Pull request merged**